src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1499,6 +1518,14 @@ public void trackNotification(final String eventName, final InAppNotification no
         }
 
         @Override
+        public void joinExperimentIfAvailable() {
+            final JSONArray variants = mDecideMessages.getVariants();
```

Comment:
That's true - the user gets one variant from every experiment, but there could be multiple experiments in progress, and this JSONArray will contain changes for all of them. (I'm using `variants` because it's used in the API)